File: python/q7.py
str = "hello"
n = len(str)
str_list = list(str)
for i in range(n // 2):
    temp = str_list[i]
    str_list[i] = str_list[n - i]
    str_list[n - i] = temp
str = "".join(str_list)
print(str)

File: C/q7.c
#include <stdio.h>
#include <string.h>
int main()
{
    char str[] = "hello";
    int n = strlen(str);
    for (int i = 0; i < n / 2; i++)
        
        {
            char temp = str[i];
            str[i] = str[n - i];
            str[n - i] = temp;
            
        }
    printf("%s\n", str);
    return 0;
}

